prog/postgres/postgres_server_nexus.rb

```diff
@@ -428,6 +437,7 @@ def before_run
   end
 
   label def taking_over
+    decr_promote
```

Comment:
you sure you want to mix this side effect with failure (rollback) -prone I/O? I mean, it may be *okay*-ish but, pointing it out.